biodatasets/epicqa/epicqa.py

```diff
@@ -0,0 +1,295 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import json
+
+import datasets
+
+from utils import schemas
+from utils.constants import Tasks
+from utils.configs import BigBioConfig
+
+_CITATION = """\
+Unknown
+"""
+
+_DESCRIPTION = """\
+In response to the COVID-19 pandemic, the Epidemic Question
+Answering (EPIC-QA) track challenges teams to develop systems
+capable of automatically answering ad-hoc questions about the
+disease COVID-19, its causal virus SARS-CoV-2, related corona
+viruses, and the recommended response to the pandemic.
+"""
+
+_DATASETNAME = "EpicQA"
+
+_HOMEPAGE = "https://bionlp.nlm.nih.gov/epic_qa/#collection"
+
+_LICENSE = "DUA-NC"
+
+_BASE = "https://bionlp.nlm.nih.gov/epic_qa/data/epic_qa_"
+
+_URLs = {
+    "epicqa_research_articles_source": _BASE + "cord_2020-10-22-split-corrected.tar.gz",
+    "epicqa_research_articles_bigbio_qa": _BASE + "cord_2020-10-22-split-corrected.tar.gz",
+    
+    "epicqa_consumer_articles_source": _BASE + "consumer_2020-11-02.tar.gz",
+    "epicqa_consumer_articles_bigbio_qa": _BASE + "consumer_2020-11-02.tar.gz",
+}
+
+_DIRS = {
+    "research_articles" : [
+        "2020-10-22-new/",
+    ],
+    "consumer_articles" : [
+        "epic_qa_consumer_2020_11-02/ask_science-2020-10-29/",
+        "epic_qa_consumer_2020_11-02/ccns-trec/",
+        "epic_qa_consumer_2020_11-02/chqa-2020-10-09/",
+    ],
+}
+
+_SUPPORTED_TASKS = [Tasks.QUESTION_ANSWERING]
+_SOURCE_VERSION = "1.0.0"
+_BIGBIO_VERSION = "1.0.0"
+
+class EpicQaDataset(datasets.GeneratorBasedBuilder):
+    """EpicQA"""
+
+    DEFAULT_CONFIG_NAME = "source"
```

Comment:
this should match one of the `name` attributes from one of the config below (e.g. `epicqa_research_articles_source`)